### PR TITLE
Break out project column mapping to own config

### LINKED ISSUE #10: Break out header names to config file
To avoid having to change the code if the headers in the source file change, e.g. "Område".

### PR DESCRIPTION
**Pull request by lokal-profil**
fixes #10

### CODE REVIEW

File: project_start.py [hunk 1]
```diff
@@ -185,6 +195,11 @@ def add_phab_project(project_information):
         help="Config file.",
         default="config.yaml"
     )
+    parser.add_argument(
```
Comment (sebastian-berlin-wmse) [inline project_start.py]: This doesn't do anything.

File: project_start.py [hunk 2]
```diff
@@ -115,45 +116,54 @@ def get_goal_name(description):
...
     """
     logging.info("Adding Phabricator project.")
-    name = project_information["Engelskt projektnamn"]
-    description = project_information["About the project"]
+    name = project_information[project_columns["english_name"]]
+    description = project_information[project_columns["about_english"]]
     return phab.add_project(name, description)
 
+
+def process_project():
```
Comment (sebastian-berlin-wmse) [inline project_start.py]: This doesn't do anything.

**Review by sebastian-berlin-wmse — CHANGES_REQUESTED** [review]
Looks good apart from a bit of dead code. Guessing it's from #17.

File: project_start.py (hunk 2 above)
Comment (lokal-profil) [reply project_start.py]: Snuck in from #17 somehow. Nuking it

File: project_start.py (hunk 1 above)
Comment (lokal-profil) [reply project_start.py]: Snuck in from #17 somehow. Nuking it

**Pull request merged**